docs/qiita-16S-analysis.rst

```diff
@@ -169,7 +169,7 @@ Faith's Phylogenetic Diversity Index
 
 The final alpha diversity analysis in this tutorial uses Faith’s phylogenetic diversity index. This index also measured abundance and diversity but considers the phylogenetic distance spanning all features in a sample. The results can also be displayed as a phylogeny, rather than as a plot.
 
-To perform an alpha diversity analysis using Faith's phylogenetic diversity index, select the "rarefid table (BIOM)" artifact in the processing network and select "Process". Select "Alpha diversity (phylogenetic)" from the drop-down menu. The parameters will appear below the workflow diagram:
+To perform an alpha diversity analysis using Faith's phylogenetic diversity index, select the "rarefied table (BIOM)" artifact in the processing network and select "Process". Select "Alpha diversity (phylogenetic)" with the Qiime2 command of "alpha phylogenetic" not "alpha phylogenetic old" from the drop-down menu. The parameters will appear below the workflow diagram:
```

Comment:
Is this right? Where did you see  `alpha phylogenetic old`?

Reply:
![image](https://user-images.githubusercontent.com/50464498/87472809-63b91000-c5d5-11ea-9707-3032c292b96e.png)
When I click on the rarefied table and select process, here's a screenshot with the alpha phylogenetic old highlighted.  Should it not be there?

Reply:
Got it, thank you for clarifying!